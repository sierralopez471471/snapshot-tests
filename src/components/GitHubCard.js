import React from 'react'
import { Card } from 'react-bootstrap'
import { Container } from 'react-bootstrap'

export default function GitHubCard() {
    return (
        <Container style={{backgroundColor: "brown", height: "100vh"}}>
            <Card style={{backgroundColor: "lightpink", borderTop: "solid 3px", borderBottom: "solid 3px", position: "relative", top: "200px"}}>
                <Card.Title style={{textAlign: "center"}}><b>sierralopez471471</b></Card.Title>
                <Card.Img src="https://avatars.githubusercontent.com/u/104409009?v=4" alt="lizard wizard" style={{border: "solid 4px", borderRadius: "15px 15px"}}>
                </Card.Img>
                <Card.Text style={{textAlign: "center", justifyContent: "space-between"}}>I'm an aspiring Full Stack Software Developer. I love horror films, psych/prog rock, and coding!</Card.Text>
            </Card>
        </Container>
    )
}