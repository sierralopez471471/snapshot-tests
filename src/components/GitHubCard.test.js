import renderer from 'react-test-renderer'
import GitHubCard from './GitHubCard.js'

test('renders a snapshot', () => {
    const tree = renderer.create(<GitHubCard/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
