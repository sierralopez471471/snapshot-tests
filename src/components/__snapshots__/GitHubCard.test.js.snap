// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="container"
  style={
    Object {
      "backgroundColor": "brown",
      "height": "100vh",
    }
  }
>
  <div
    className="card"
    style={
      Object {
        "backgroundColor": "lightpink",
        "borderBottom": "solid 3px",
        "borderTop": "solid 3px",
        "position": "relative",
        "top": "200px",
      }
    }
  >
    <div
      className="card-title h5"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <b>
        sierralopez471471
      </b>
    </div>
    <img
      alt="lizard wizard"
      className="card-img"
      src="https://avatars.githubusercontent.com/u/104409009?v=4"
      style={
        Object {
          "border": "solid 4px",
          "borderRadius": "15px 15px",
        }
      }
    />
    <p
      className="card-text"
      style={
        Object {
          "justifyContent": "space-between",
          "textAlign": "center",
        }
      }
    >
      I'm an aspiring Full Stack Software Developer. I love horror films, psych/prog rock, and coding!
    </p>
  </div>
</div>
`;
